ibexa/discounts/4.6/config/routes/ibexa_discounts.yaml

```diff
@@ -0,0 +1,6 @@
+ibexa.discounts.rest:
```

Comment:
`ibexa/discounts/4.6/config/routes/ibexa_discounts.yaml` is that correct? I don't see any indication that this package should be installed only for `commerce` tier.

Reply:
Recipes are unaware of access tiers. They either are applied due to a package presence, or they are not.

If you have the `ibexa/discounts` package installed, it's recipe will be applied by Symfony's Composer plugin.